pkg/config/config_template.yaml

```
@@ -2874,15 +2874,141 @@ api_key:
 ###################################
 
 ## @param otlp_config - custom object - optional
-## This section configures the OpenTelemetry OTLP receiver.
+## This section configures OTLP ingest in the Datadog Agent.
 #
 # otlp_config:
 
   ## @param receiver - custom object - optional
-  ## The receiver configuration. It is an exact copy of the current OpenTelemetry Collector Receiver Configuration:
-  ## https://github.com/open-telemetry/opentelemetry-collector/blob/47a899d/receiver/otlpreceiver/config.md
-  ## Because this configuration is marked as "Beta" by OpenTelemetry, it is subject to change. See also:
-  ## https://github.com/open-telemetry/opentelemetry-collector#status.
+  ## The receiver configuration. It follows the OpenTelemetry Collector's OTLP Receiver Configuration.
+  ## This template lists the most commonly used settings; see the OpenTelemetry Collector documentation 
+  ## for a full list of available settings:
+  ## https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/otlpreceiver/config.md
   #
   # receiver:
-  #   <RECEIVER_CONFIGURATION>
+
+    ## @param protocols - custom object - optional
+    ## Configuration for the supported protocols.
+    #
+    # protocols:
+
+        ## @param grpc - custom object - optional
+        ## Configuration for OTLP/gRPC listener.
+        ## Setting this as an empty section enables the OTLP/gRPC listener with default options.
+        #
+        # grpc:
+
+          ## @param endpoint - string - optional - default: 0.0.0.0:4317
+          ## @env DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT - string - optional - default: 0.0.0.0:4317
+          ## The OTLP/gRPC listener endpoint.
+          # 
+          # endpoint: 0.0.0.0:4317
+
+          ## @param transport - string - optional - default: tcp
+          ## @env DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_TRANSPORT - string - optional - default: tcp
+          ## The OTLP/gRPC listener transport protocol. 
+          ## Known protocols are "tcp", "udp", "ip", "unix", "unixgram", and "unixpacket".
+          # 
+          # transport: tcp
+
+        ## @param http - custom object - optional
+        ## Configuration for OTLP/HTTP listener.
+        ## Setting this as an empty section enables the OTLP/HTTP listener with default options.
+        #
+        # http:
+
+          ## @param endpoint - string - optional - default: 0.0.0.0:4318
+          ## @env DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT - string - optional - default: 0.0.0.0:4318
+          ## The OTLP/HTTP listener endpoint.
+          # 
+          # endpoint: 0.0.0.0:4318
+
+  ## @param metrics - custom object - optional
+  ## Metrics-specific configuration for OTLP ingest in the Datadog Agent.
```

Since the config option may change in the future, let's call that out, for example (more details would be welcome though):

```suggestion
  ## Metrics-specific configuration for OTLP ingest in the Datadog Agent.
  ## Note that these configuration options may change in future minor Agent versions.
```